stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/internal/MessageListItemViewTypeMapper.kt

```diff
@@ -60,6 +62,7 @@ internal object MessageListItemViewTypeMapper {
             containsGiphy -> GIPHY_ATTACHMENT
             containsOnlyLinks -> LINK_ATTACHMENTS
             message.isImageAttachment() -> IMAGE_ATTACHMENT
+            hasAttachments -> FILE_ATTACHMENTS
             message.attachments.isNotEmpty() -> TEXT_AND_ATTACHMENTS
```

Comment:
Hm, basically we have the following now:           
```
hasAttachments -> FILE_ATTACHMENTS
hasAttachments -> TEXT_AND_ATTACHMENTS
```
But that's fine for now, I'll fix this when after refactoring to handlers.

Reply:
Yes, sounds good!